spec/solver_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe Solver do
+  before(:each) do
+    @solver = Solver.new
+  end
+  it 'Factorial of 5 must equal 120' do
+    expect(@solver.factorial(5)).to be 120
+  end
+
```

Comment:
- [ ] Kindly ensure to include all tests for the factorial method, below is a screenshot highlighting all the tests required
![image](https://user-images.githubusercontent.com/61408860/201134684-7dcdc4ac-e3ee-4630-a394-2654fc03406c.png)

hint ` for easier readability you could separate the test cases using a describe within the current describe so as to separate the tests for factorial, fizzbuzz, and reverse methods.`